### DISCUSSION TITLE
Not able to view balance and UTXO of an account

### DISCUSSION THREAD

**Discussion by rdx4499** [1200wd/bitcoinlib#397] [Q&A] [answered]
I have created one account with address : tb1qg4jfeunmfkkw4ydmslaje6tu0a8p67zaxv8dct
I have sent some test BTC to this address : https://blockstream.info/testnet/address/tb1qg4jfeunmfkkw4ydmslaje6tu0a8p67zaxv8dct
you can see UTXO on the explorer, now I want to sweep this BTC with other account but getting error for not having any UTXO, I have tried out transaction_update, utxo_update, scan() but none of them update my balance 
And when I do wallet.info() this is what its print 

=== WALLET ===
 ID                             1
 Name                           test1_1714130365
 Owner                          
 Scheme                         bip32
 Multisig                       False
 Witness type                   segwit
 Main network                   testnet
 Latest update                  2024-04-29 13:03:18.282727

= Wallet Master Key =
 ID                             1
 Private                        True
 Depth                          0

- NETWORK: testnet -
- - Keys
    6 m/84'/1'/0'/0/0              tb1qnttduzyhd0gmu2ve4wzthtyqmx9rrxh4fd5tv7    address index 0                     0.00000000 TBTC
    7 m/84'/1'/0'/0/1              tb1q5da0x9sstr4l8803m4zcn27kqazwzkyrwftqef    address index 1                     0.00000000 TBTC
    8 m/84'/1'/0'/0/2              tb1qdxkgq4806gwld8a7eflhtk2y0srvruu637akl0    address index 2                     0.00000000 TBTC
    9 m/84'/1'/0'/0/3              tb1qpfkepx5pr5hkyatmhft58ft83h593la8967gwy    address index 3                     0.00000000 TBTC
   10 m/84'/1'/0'/0/4              tb1qefel5m38lx3vl0kj59py0y4hd2hvxun06vw7fg    address index 4                     0.00000000 TBTC
   12 m/84'/1'/0'/1/0              tb1qy87d5lgsfna63zd4fw8hae82t4nu6kxga4355d    address index 0                     0.00000000 TBTC
   13 m/84'/1'/0'/1/1              tb1q7u5xs8dn8slsx9240nkh3pxxptyrr4lvdr0nm4    address index 1                     0.00000000 TBTC
   14 m/84'/1'/0'/1/2              tb1q9hm78kkawlykcfyd7ggzccgfqglezg7m7pal9d    address index 2                     0.00000000 TBTC
   15 m/84'/1'/0'/1/3              tb1q4wckl464pqrls3n2qkure6pupcxw930x6al9rg    address index 3                     0.00000000 TBTC
   16 m/84'/1'/0'/1/4              tb1q7hv438f5rpcq3h037qx84lm0k0h4j43yzpncaz    address index 4                     0.00000000 TBTC

- - Transactions Account 0 (3)
5e99d1ee4afbdfd303f70ee9671c63eec66d2efe65413d1985c88f8973754156  tb1qg4jfeunmfkkw4ydmslaje6tu0a8p67zaxv8dct    11694       0.00002000 TBTC   
f76f95865615caa5e789c36c1b42932c7ab71b9144ebc62961efb3ddebc7d418  tb1qg4jfeunmfkkw4ydmslaje6tu0a8p67zaxv8dct    10071       0.00001000 TBTC   
632d2972812bd1a98e154ed806238d54ee6699883c01486a411b706395cd54e4  tb1qg4jfeunmfkkw4ydmslaje6tu0a8p67zaxv8dct     9928       0.00001000 TBTC   

= Balance Totals (includes unconfirmed) =

As you can see there is no balance or UTXO in the output.

**Comment by mccwdev** [comment, marked as answer]
There are probably errors when connecting to some service providers. If you check the logs in .bitcoinlib/bitcoinlib.log you can find the specific errors and see if you can solve them.

Also for some providers it can take a couple of minutes before utxo's are updated. So retrying after a short while could also help.
